src/Exceptions/HubspotException.php

```diff
@@ -2,6 +2,38 @@
 
 namespace SevenShores\Hubspot\Exceptions;
 
-class HubspotException extends \Exception
+use Exception;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7\Response;
+
+class HubspotException extends Exception
 {
+    /** @var Response|null */
+    protected $response;
+
+    /**
+     * @return Response|null
+     */
+    public function getResponse()
+    {
+        return $this->response;
+    }
+
+    public static function create(RequestException $guzzleException): self
+    {
+        $e = new static(
+            self::sanitizeResponseMessage($guzzleException->getMessage()),
```

Comment:
Could You change self to static at this place?
I understand that won’t  be redefined but it is our style
over time we will change all private to protected and self to static (where it is necessary)